modules/components/src/Aggs/BooleanAgg.js

```diff
@@ -58,13 +75,12 @@ export default ({
     });
 
   return (
-    <AggsWrapper {...{ buckets, ...rest }}>
+    <AggsWrapper {...{ displayName, WrapperComponent, collapsible }}>
       <div className={`booleanFacetWrapper`}>
         <div
```

Comment:
If you feel like it I suppose this could be the second use of `ToggleButton`

Reply:
updated BooleanAgg to render using the ToggleButton :) thanks!